src/aerovaldb/utils/typing.py

```diff
@@ -0,0 +1,21 @@
+import sys
```

Comment:
rename `utils/typing.py` to `compat/typing.py` since it is only a compatibility layer.